inc/experiments/namespace.php

```diff
@@ -483,13 +486,15 @@ function enqueue_experiments_editor_scripts( string $hook ) : void {
 		foreach ( $test['editor_scripts'] as $script => $deps ) {
 			wp_enqueue_script( "altis-experiments-features-{ $test_id }", $script, array_merge( $deps, [ 'altis-experiments-features' ] ), null );
 		}
-	}
 
-	$js_data = array_filter( array_map( function( array $test ) {
-		return $test['show_ui']
-			? array_intersect_key( $test, array_flip( [ 'label', 'singular_label' ] ) )
-			: false;
-	}, $post_ab_tests ) );
+		if ( $test['show_ui'] ) {
+			$js_data[$test_id] = [
```

Comment:
Minor issue on the formatting, we use spaces around array keys if it's a variable:
```suggestion
			$js_data[ $test_id ] = [
```

Reply:
Thanks for catching that @roborourke , I had hoped to run the HM Linter before submitting the code review but it wasn't showing.